docs/TESTS.md

```diff
@@ -1,112 +1,65 @@
-## Xcode Tutorial
+## Testing Tutorial
 
-This guide explains how to create and configure a Swift-based Xcode project for Exercism.
+This guide explains how to run unit tests for Exercism Swift exercises, and optionally how to use Xcode for development.
 
-Apple does not support user-defined project templates in Xcode, and the default templates are more elaborate than those needed for Exercism. This guide bridges that gap.
+Exercism uses [Swift Package Manager](https://github.com/apple/swift-package-manager/tree/master/Documentation) to package files and tests for Swift.
```

Comment:
I think adding the Swift Package Manager is a big improvement. But I'm concerned the SPM edits made to Tests.md drops some important concepts that users who are coming in with a "beginners mindset" may find helpful. For example:

The context isn't clear that there are now two techniques for running the tests, Xcode and command line. Nor the consequences of each option. For example, the loss of inline compiler messages.

Running tests from the command line strips away error messages that might help new users, the command line output only displays messages from the console, not stderr. Depending on the situation, that missing info is valuable.

Bob